test/soak/serve_hostnames/serve_hostnames.go

```diff
@@ -44,6 +44,7 @@ var (
 	podsPerNode    = flag.Int("pods_per_node", 1, "Number of serve_hostname pods per node")
 	upTo           = flag.Int("up_to", 1, "Number of iterations or -1 for no limit")
 	maxPar         = flag.Int("max_par", 500, "Maximum number of queries in flight")
+	gke            = flag.String("gke", "", "Taget GKE cluster with context gke_{project}_{zone}_{cluster-name}")
```

Comment:
It'd be more generic to make a --context flag but as a GKE user I appreciate making this particular case nice and easy.